README.md

```diff
@@ -105,7 +105,7 @@ Settings related to displaying UART output in the debug console. This object can
 
 ## Memory Browser
 
-The extension comes with a Memory Browser window. However, we have plans to deprecate and remove the window in favour of the Eclipse CDT Cloud [Memory Inspector](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector) extension for Visual Studio Code which is available from the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=eclipse-cdt.memory-inspector) and the [Open VSX Registry](https://open-vsx.org/extension/eclipse-cdt/memory-inspector).
+The extension comes with a Memory Browser window. However, we have plans to deprecate and remove the window in favour of the Eclipse CDT Cloud [Memory Inspector](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector) extension for vscode which is available from the [Microsoft Marketplace](https://marketplace.visualstudio.com/items?itemName=eclipse-cdt.memory-inspector) and the [Open VSX Registry](https://open-vsx.org/extension/eclipse-cdt/memory-inspector).
```

Comment:
Again, better `VS Code`.
I believe we should at least say `VS Code Marketplace` (with or without `Microsoft`). Microsoft has other marketplaces as well, e.g. https://azuremarketplace.microsoft.com/de-de/

Reply:
agreed